client/src/game/ui/notes.ts

```diff
@@ -104,126 +107,100 @@ export const checkSpecialNote = (card: HanabiCard) => {
   globals.layers.card.batchDraw();
 };
 
+const checkNoteKeywords = (
+  keywords: string[],
+  note: string,
+  fullNote: string,
+) => keywords.find((k) => note === k || fullNote.includes(`[${k}]`)) !== undefined;
+
 const checkNoteIdentity = (card: HanabiCard, note: string, fullNote: string) => {
   // First, check to see if this card should be marked with certain properties
-  card.state.noteKnownTrash = (
-    note === 'kt'
-    || fullNote.includes('[kt]')
-    || note === 'trash'
-    || fullNote.includes('[trash]')
-    || note === 'stale'
-    || fullNote.includes('[stale]')
-    || note === 'bad'
-    || fullNote.includes('[bad]')
-  );
-  card.state.noteNeedsFix = (
-    note === 'fixme'
-    || fullNote.includes('[fixme]')
-  );
-  card.state.noteChopMoved = (
-    note === 'cm'
-    || fullNote.includes('[cm]')
-    || note === 'chop move'
-    || fullNote.includes('[chop move]')
-    || note === '5cm'
-    || fullNote.includes('[5cm]')
-    || note === 'e5cm'
-    || fullNote.includes('[e5cm]')
-    || note === 'tcm'
-    || fullNote.includes('[tcm]')
-    || note === 'tccm'
-    || fullNote.includes('[tccm]')
-    || note === 'sdcm'
-    || fullNote.includes('[sdcm]')
-    || note === 'sbpcm'
-    || fullNote.includes('[sbpcm]')
-    || note === 'ocm'
-    || fullNote.includes('[ocm]')
-    || note === 'tocm'
-    || fullNote.includes('[tocm]')
-    || note === 'utfcm'
-    || fullNote.includes('[utfcm]')
-    || note === 'utbcm'
-    || fullNote.includes('[utbcm]')
-  );
-  card.state.noteFinessed = (
-    note === 'f'
-    || fullNote.includes('[f]')
-    || note === 'pf'
-    || fullNote.includes('[hf]')
-    || note === 'hf'
-    || fullNote.includes('[pf]')
-    || note === 'gd'
-    || fullNote.includes('[gd]')
-  );
-  card.state.noteBlank = (
-    note === 'blank'
-    || fullNote.includes('[blank]')
-  );
-  card.state.noteUnclued = (
-    note === 'unclued'
-    || fullNote.includes('[unclued]')
-  );
+  card.state.noteKnownTrash = checkNoteKeywords([
+    'kt',
+    'trash',
+    'stale',
+    'bad',
+  ], note, fullNote);
+
+  card.state.noteNeedsFix = checkNoteKeywords([
+    'fixme',
```

Comment:
small thing - make this one line, and if we add an alias we can split it into separate lines

Reply:
ok, i was wondering about those